python/test/unit/language/test_matmul.py

```diff
@@ -682,8 +682,10 @@ def block_scale_fp4_matmul(  #
     # Two e2m1 values per K
     offs_k = tl.arange(0, BLOCK_K // 2)
     offs_scale_k = tl.arange(0, BLOCK_K // VEC_SIZE)
-    a_scale_ptr = a_scale + offs_am[:, None] * stride_scale + offs_scale_k[None, :]
```

Comment:
are those existing tests enough:
https://github.com/triton-lang/triton/blob/main/python/test/unit/language/test_core.py#L3688

Reply:
That's for the scaled_dot with single mxfp operand. But this PR is for double mxfp scaled_dot, where both operands can be mxfp4 or normal fp4 if its scale is None.